modules/ts/src/ts.cpp

```diff
@@ -723,6 +723,7 @@ void checkIppStatus()
 
 bool skipUnstableTests = false;
 bool runBigDataTests = false;
+bool checkTestData = false;
```

Comment:
It is not defined in headers, so use "static".

Environment parameter is easy to integrate into build system, so this would nice to have:
```
static bool g_requireTestData = utils::getConfigurationParameterBool("OPENCV_TEST_REQUIRE_DATA", false);

...

if (parser.get<bool>("test_data_check"))
    g_requireTestData = true;
```